src/components/Media/Media.tsx

```diff
@@ -155,8 +158,24 @@ export const Media = (props: MediaAllProps) => {
         margins,
     ]);
 
+    const videoMicrodataScript = useMemo(() => {
+        const json = JSON.stringify({
+            '@context': 'http://schema.org/',
+            '@type': 'VideoObject',
+            uploadDate: microdata?.contentUpdatedDate,
+            contentUrl: video?.src?.[0] || videoIframe || youtube,
+            thumbnailUrl: previewImg,
+            ...videoMicrodata,
```

Comment:
maybe we need to use `sanitizeMicrodata` only in this place ?

Reply:
fixed